HOWTO.md

```diff
@@ -576,6 +601,25 @@ Please mind how the configuration file must be specifically named (e.g. `-` VS
 
 ## How to run the Client
 
+### `fdo-manufacturing-client`
+
+1. If you come across use case such as: you need to check DI (Device Initialize)
```

Comment:
I would completely delete this first step, we are in the HOWTO, not in the CONTRIBUTING guide, the info needs to be objective